### PR TITLE
Create 2018-11-26-Square-CTF-2018-Gate-of-hell.md

### PR DESCRIPTION
**Pull request by Corallo**
New writeup from Square CTF

### CODE REVIEW

File: _posts/2018-11-26-Square-CTF-2018-Gate-of-hell.md [hunk 1]
```diff
@@ -0,0 +1,202 @@
+---
+title: Square CTF 2018 - Gates of hells WriteUp
+layout: post
+author: "XxcoralloxX"
+tags: reverse TuCTF
```
Comment (suidpit) [inline _posts/2018-11-26-Square-CTF-2018-Gate-of-hell.md]: Is this relative to TuCTF or Square CTF? 👅

File: _posts/2018-11-26-Square-CTF-2018-Gate-of-hell.md [hunk 2]
```diff
@@ -0,0 +1,202 @@
...
+2) have a better look at the instruction at point 3) which could make ebx to reset (and we will lose)
+
+3) Choose the index of the vector, so that their product will be equal to 18 (since ebx start to 37 (0x25) 37*18 = 666 (0x29A))
+
+
+## Task 1
+The first task is quite easy.
+
+Since 
+resoult=(ch1-0x30)*0xa+(ch2-0x30);
```
Comment (suidpit) [inline _posts/2018-11-26-Square-CTF-2018-Gate-of-hell.md:120]: You will need to escape the * to preserve Markdown consistency!

File: _posts/2018-11-26-Square-CTF-2018-Gate-of-hell.md [hunk 3]
```diff
@@ -0,0 +1,202 @@
...
+So we need to provide 15 arguments? (the first is the name of the program).
+
+Ok, 15 arguments, but when will we win?
+we can find this function
+
+![AltText](https://i.gyazo.com/93807576fe368fb6a98601cfc369db6f.png)
+
+which is a system call to a print. (which is probably our win function).
+
+as we can see this function will be called if Ebx will be equal at 0x29A (666)
```
Comment (suidpit) [inline _posts/2018-11-26-Square-CTF-2018-Gate-of-hell.md]: One thing that could be useful for the next writeups is enclosing register names in a rich text format, like `ebx`, _ebx_ or **ebx**

**Review by suidpit — COMMENTED** [review]
Left some comments! Take a look :)

**Comment by Corallo** [comment]
Just fixed tag name, and escaped all the '*'

**Pull request merged**